musica/mechanism_configuration/condensed_phase_arrhenius.py

```diff
@@ -63,59 +63,246 @@ def __init__(
+        self._other_properties = value
+        # Update the C++ instance
+        self._instance.other_properties = value
+
+    def serialize(self) -> Dict:
+        """
+        Serialize the CondensedPhaseArrhenius instance to a dictionary.
+        
+        Returns:
+            Dict: A dictionary representation of the condensed phase Arrhenius rate constant.
+        """
+        # Convert Python reactants/products to serializable format
+        def serialize_python_components(components):
+            result = []
+            for component in components:
+                if isinstance(component, Species):
+                    result.append(component.name)
+                elif isinstance(component, tuple) and len(component) == 2:
+                    # Handle (coefficient, Species) tuples
+                    coefficient, species = component
+                    result.append({
+                        "species name": species.name,
+                        "coefficient": coefficient
+                    })
+                else:
+                    # Fallback: treat as Species
+                    result.append(component.name if hasattr(component, 'name') else str(component))
+            return result
+        
         serialize_dict = {
             "type": "CONDENSED_PHASE_ARRHENIUS",
-            "name": instance.name,
-            "A": instance.A,
-            "B": instance.B,
-            "C": instance.C,
-            "D": instance.D,
-            "E": instance.E,
-            "reactants": ReactionComponentSerializer.serialize_list_reaction_components(instance.reactants),
-            "products": ReactionComponentSerializer.serialize_list_reaction_components(instance.products),
-            "aerosol phase": instance.aerosol_phase,
-            "aerosol-phase water": instance.aerosol_phase_water,
+            "name": self.name,
+            "A": self.A,
+            "B": self.B,
+            "C": self.C,
+            "D": self.D,
+            "E": self.E,
+            "reactants": serialize_python_components(self._reactants),
+            "products": serialize_python_components(self._products),
+            "aerosol phase": self._aerosol_phase.name if self._aerosol_phase is not None else "",
+            "aerosol-phase water": self._aerosol_phase_water.name if self._aerosol_phase_water is not None else "",
         }
-        _add_other_properties(serialize_dict, instance.other_properties)
+        _add_other_properties(serialize_dict, self._other_properties)
         return _remove_empty_keys(serialize_dict)
+
+    @staticmethod
+    def serialize_static(instance) -> Dict:
+        """
+        Static serialize method for backward compatibility.
+        
+        Args:
+            instance: The CondensedPhaseArrhenius instance to serialize.
+            
+        Returns:
+            Dict: A dictionary representation of the condensed phase Arrhenius rate constant.
+        """
+        return instance.serialize()
```

Comment:
This will only work if the type is a python type. This serialize_static method is only called with the c++ wrapper types, an `_CondensedPhaseArrhenius`